src/System.Runtime.Extensions/tests/System/Environment.SetEnvironmentVariable.cs

```diff
@@ -61,6 +61,7 @@ public void IncorrectVariableThrowsArgument()
         }
 
         [Theory]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot, "Environment Variables monidification isn't supported in Appx.")]
```

Comment:
It seems very important to have an issue which lists all tests that are disabled because of different behaviors on uap/uapaot. I will do that and link to all occurences.